lms/templates/vert_module.html

```diff
@@ -61,3 +67,9 @@ <h2 class="hd hd-2 unit-title">${unit_title}</h2>
 <%static:require_module_async module_name="js/dateutil_factory" class_name="DateUtilFactory">
     DateUtilFactory.transform('.localized-datetime');
 </%static:require_module_async>
+
+<script>
+    function emit_event(message) {
+        parent.postMessage(message, '*');
```

Comment:
Maybe we should give an event name up to the parent, like "submit_event" or something, so the MFE knows how to interpret the dictionary we give it (i.e. it knows to look for the fields it needs to run a POST).

Just in case we ever want to add a new kind of iframe message.